src/ResourceManager/AnalysisServices/Commands.AnalysisServices/Commands/NewAzureRmAnalysisServicesServer.cs

```diff
@@ -41,10 +41,7 @@ public class NewAnalysisServicesServer : AnalysisServicesCmdletBase
         [Parameter(ValueFromPipelineByPropertyName = true, Position = 2, Mandatory = true,
             HelpMessage = "Azure region where the server should be created.")]
         [ValidateNotNullOrEmpty]
-        [ValidateSet("North Central US", "South Central US", "Central US", "West Europe", "North Europe", "West US",
-            "East US",
-            "East US 2", "Japan East", "Japan West", "Brazil South", "Southeast Asia", "East Asia", "Australia East",
-            "Australia Southeast", IgnoreCase = true)]
+        [LocationCompleter("Microsoft.AnalysisServices")]
```

Comment:
The LocationCompleter requires both the provider and resource type - I think here it would require "Microsoft.AnalysisServices/servers"

Reply:
I tested it on prod cloud and gov cloud, it seemed working(at least I can provision a server and fail test works as well). Let me try adding the resource type.

Reply:
Both "Microsoft.AnalysisServices/servers" and "Microsoft.AnalysisServices" work, since our code is resource type specific, I will update it as "Microsoft.AnalysisServices/servers".